src/Tools/MicroBuild/run-pdbgit.ps1

```diff
@@ -0,0 +1,84 @@
+<#
+
+    Run PDB Git on our PDB files to enable proper source server support
+
+#>
+Param(
+    [string]$sourcePath = $null,
+    [string]$binariesPath = $null
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+function Get-PackagesPath {
+    $packagesPath = $env:NUGET_PACKAGES
+    if ($packagesPath -eq $null) {
+        $packagesPath = join-path $env:UserProfile ".nuget\packages\"
+    }
+
+    return $packagesPath
+}
+
+try
+{
+    [xml]$deps = get-content (join-path $sourcePath "build\Targets\Dependencies.props")
+    $pdbGitVersion = $deps.Project.PropertyGroup.PdbGitVersion
+    $packagesPath = Get-PackagesPath
+    $pdbGit = join-path $packagesPath "pdbgit\$pdbGitVersion\tools\PdbGit.exe"
+
+    # Linking our PDBs to Git is really only interesting for PE / PDB that are 
+    # being shipped to customers.  For local developement the original PDB 
```

Comment:
Ya, that's where I thought it would be. At the time I wrote that, I knew it was horrifically poor, but I couldn't find a more efficient way to "walk the tree" to normalize capitalization (which is very important, it turns out). If you know the more efficient way, yes please do send a PR. :)